machines/Makefile

```diff
@@ -15,16 +15,18 @@ VCS_INCLUDES += $(foreach inc,$(VINCLUDES),+incdir+"$(inc)")
 VCS_DEFINES  += $(foreach def,$(VDEFINES),+define+"$(def)")
 VCS_SOURCES  += $(VHEADERS) $(VSOURCES) 
 VCS_FLAGS    += +v2k -sverilog -full64 -timescale=1ps/1ps \
-	 	+lint=all,noSVA-UA,noSVA-NSVU,noVCDE,noNS
+	 	+lint=all,noSVA-UA,noSVA-NSVU,noVCDE,noNS -top spmd_testbench
 
-VCS_CFLAGS   =  -cpp clang++
+VCS_CFLAGS   =  -cpp g++
```

Comment:
Just generally curious why clang was necessary at one point. Not relevant to feedback on this PR. Did it work?

Reply:
ramulator had some c++ stuff that required it.